### PR TITLE
[BACKLOG-14320] Adding VizAPI sample files

### PR DESCRIPTION
**Pull request by dcleao**
If and when this is accepted, will remove these files from the @nantune's viz-sandbox repo.

@pentaho/millenniumfalcon please review.

### CODE REVIEW

File: Samples for Extending Pentaho/javascript-apis/pentaho/visual/bar-d3-sandbox/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "pentaho-visual-samples-bar-d3",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Pentaho Visual Samples D3-Bar",
+  "license": "MIT",
+  "dependencies": {
+    "@nantunes/viz-api": "^3.0.2",
```
Comment (nantunes) [inline Samples for Extending Pentaho/javascript-apis/pentaho/visual/bar-d3-sandbox/package.json]: This is not final (obvious, but just to highlight it).

File: Samples for Extending Pentaho/javascript-apis/pentaho/visual/bar-d3-sandbox/index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    .pentaho-visual-base {
+      border: solid 1px #005da6;
+    }
+  </style>
+
+  <!-- load requirejs -->
+  <script type="text/javascript" src="node_modules/requirejs/require.js"></script>
+
+  <!-- load the VizAPI dev bootstrap helper -->
+  <script type="text/javascript" src="node_modules/@nantunes/viz-api/dev-bootstrap.js"></script>
```
Comment (nantunes) [inline Samples for Extending Pentaho/javascript-apis/pentaho/visual/bar-d3-sandbox/index.html]: Again, this is not final (obvious, again).

File: Samples for Extending Pentaho/javascript-apis/pentaho/visual/sandbox/package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "pentaho-viz-sandbox",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Pentaho Visualization API sandbox",
+  "license": "MIT",
+  "author": "Nelson Antunes <nantunes@pentaho.com>",
+  "dependencies": {
+    "@nantunes/viz-api": "^3.0.2"
```
Comment (nantunes) [inline Samples for Extending Pentaho/javascript-apis/pentaho/visual/sandbox/package.json]: Again, this is not final (did I say it is obvious?).

File: Samples for Extending Pentaho/javascript-apis/pentaho/visual/sandbox/index.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    .pentaho-visual-base {
+      border: solid 1px #005da6;
+    }
+  </style>
+
+  <!-- load requirejs -->
+  <script type="text/javascript" src="node_modules/requirejs/require.js"></script>
+
+  <!-- load the VizAPI dev bootstrap helper -->
+  <script type="text/javascript" src="node_modules/@nantunes/viz-api/dev-bootstrap.js"></script>
```
Comment (nantunes) [inline Samples for Extending Pentaho/javascript-apis/pentaho/visual/sandbox/index.html]: Obviously this is not final (...).

**Review by nantunes — APPROVED** [review]

**Comment by dcleao** [comment]
@pamval could you please merge this, as we at @pentaho/millenniumfalcon don't have permissions.

**Comment by pamval** [comment]
@dkincade, I can't merge as well.  please merge when you have the chance.

**Pull request merged**